### DISCUSSION TITLE
Tab font size is too small

### DISCUSSION THREAD

**Discussion by ig1727** [freeplane/freeplane#1654] [Help]
I am using version 1.7.10 in Ubuntu Linux.
The font size for tab captions is so small that it is hardly readable.
Any suggestions on how to adjust it (make it bigger)?

**Comment by euu2021** [comment]
You can try `Preferences…->Appearance->Look and feel->Menu item font size` other Look and Feel settings:
![image](https://github.com/freeplane/freeplane/assets/77707706/591482e2-8e82-4ea1-b476-94b551175613)

**Reply by ig1727** [reply]
Unfortunately, changing the menu font size doesn't change font size for tabs.
If several maps exist, the tab font size is not changing by this parameter.
I played with other parameters but could not find a way how to make the font bigger:

![image](https://github.com/freeplane/freeplane/assets/146740824/bece81cd-9ef1-4975-93a7-4ab30484b784)

On this snapshot, the font looks reasonably large, but on the monitor, it seems like a size six or less.

On Windows, the font for tabs is sized normally, but on Ubuntu Linux, with the Wayland display manager doesn't.
Maybe I can change deep settings in the configuration files?

**Reply by dpolivaev** [reply]
On Linux, there is a preference for "monitor size". It is not available on other OS where Java can get the monitor scaling automatically. It is responsible for scaling all fonts used by Freeplane. Usually, if people use it correctly all fonts are nicely readable.

**Reply by AlexandreBonneau** [reply]
I can confirm that using Freeplane on Linux (Debian) on a 4k screen is perfectly readable:
<img width="542" alt="2024-02-05_164113_Screenshot" src="https://github.com/freeplane/freeplane/assets/7248877/5b1c8238-300c-44ad-8dd7-40dc7d8d2d1a">
